### PR TITLE
docs: update docs with new features

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR updates the documentation of the plugin including the new features.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -151,10 +203,96 @@ display add the new tab for On Task:
...
+will be named as the current course ID. e.g. ``course-v1:edunext+ontask+demo``.
+
+.. image:: https://github.com/eduNEXT/platform-plugin-ontask/assets/64033729/ab2c2623-1b74-479e-b8f2-7d37cb4b9a64
+
+The instructor can then view the OnTask workflow related to the course. From
+there the instructor can create, import/export or execute all actions. In the
+upper left corner there will be a button called **Load Data**. Once pressed, an
+asynchronous task will be executed that will generate a data summary of the
+course and load it into a table in the OnTask Workflow. Each time the button is
+pressed, the data will be overwritten. The instructor can then use this data to
```
Comment (mariajgrimaldi) [inline README.rst]: Is _Each time the button is pressed, the data table will be recreated_ a better description?

File: README.rst [hunk 2]
```diff
@@ -151,10 +203,96 @@ display add the new tab for On Task:
...
+that is loaded into the OnTask table. To do this, follow these steps:
+
+1. Create a new file in the ``datasummary`` directory with the name of the
+   backend, e.g., ``custom.py``
+2. Create a class that inherits from ``DataSummary``, e.g.,
+   ``CustomDataSummary(DataSummary)``
+3. Implement the ``get_data_summary`` method to return the data summary. The
+   method must return a dictionary where each key is the column name and the
+   value is other dictionary with the id as key and the value as the value of
+   the column. You can check an example in the ``dummy.py`` file.
```
Comment (mariajgrimaldi) [inline README.rst]: Great! Can we add an example of how this would look like?

**Review by mariajgrimaldi — APPROVED** [review]
Minor comments :) Thanks!

File: README.rst (hunk 1 above)
Comment (BryanttV) [reply README.rst]: Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/69d1b4881df347721a718e87f6ee1ff68c598001

File: README.rst (hunk 2 above)
Comment (BryanttV) [reply README.rst]: Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/dc607bd67737935a5987f02caefa61875f4dfb6b

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**